src/main/java/ai/zuva/docai/DocAIClient.java

```diff
@@ -72,6 +75,22 @@ private static MediaType toMediaType(String... contentType) {
     return mediaType;
   }
 
+  public Map<String, String> listToMapQueryParams(List<String> requestIds) {
```

Comment:
We might as well make this more general, like `listToMapQueryParams(String key, List<String> values)`, since we also have the multiple delete endpoint with the same pattern for `file_id`.

Reply:
b3b05a14f2d42aa35f25aefa8cc460b1755f364b